lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Hm, so you could just do

```ruby
form_for "" do |f|
  f.text_field :foo
  f.select :bar
end
```

Or do you mean override the `form_tag` helper as well somehow? I guess that would be weird since it would change its API..